pkg/cluster/nodes/create.go

```diff
@@ -142,6 +151,8 @@ func createNode(name, image, clusterLabel, role string, mounts []cri.Mount, extr
 		"--label", clusterLabel,
 		// label the node with the role ID
 		"--label", fmt.Sprintf("%s=%s", constants.NodeRoleKey, role),
+		// connect node to network
+		"--network", clusterName,
```

Comment:
what are valid values for this? does clusterName and network name really have the same restrictions? EG for containers we always append something so it will pass validation differently

Reply:
it only need `string` https://github.com/moby/moby/blob/master/api/types/types.go#L475